### DISCUSSION TITLE
Third-party content of NOTICE.md files

### DISCUSSION THREAD

**Discussion by bhufmann** [eclipse-cdt-cloud/cdt-cloud#15] [Q&A]
Hi,

I'd like to add a NOTICE.md file to the new tracing related repositories (e.g. [theia-trace-extension](https://github.com/eclipse-cdt-cloud/theia-trace-extension)). I'm reading the [guidelines](https://www.eclipse.org/projects/handbook/#legaldoc-notice) from Eclipse.org. I'm not clear what to put in the `Third-party content` section. I looked at examples ([cdt-cloud-blueprint](https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint/blob/master/NOTICE.md) or [theia-core](https://github.com/eclipse-theia/theia/blob/master/NOTICE.md)).  Clearly that list are not all dependencies that are in the yarn.log or found with dash-licenses. 

Does anybody know what needs to be listed there?

Moreover, how is this list generated? Is there a tool available that creates this list? Having to do this manually would be error-prone and hard to maintain. 

Please let me know. Thanks in advance for you answers.

**Comment by jonahgraham** [comment]
In the past the [Legal Documentation Generator](https://www.eclipse.org/projects/tools/documentation.php?id=ecd.cdt.cloud) would do that based on the CQ records of the project. But with CQs being relegated more and more to history I am not sure how to do that automatically anymore.

Can I recommend holding off on doing anything for now as there seems to be an IP update coming that may impact these things. Mike M will be talking at the [upcoming architecture council meeting](https://wiki.eclipse.org/Architecture_Council/Meetings/Meeting_Notes).

**Reply by bhufmann** [reply]
Interesting. Thanks a lot. I didn't know about this generator. The CQs for the repos haven't not been approved. We only got the go ahead to transfer the repo under the `eclipse-cdt-cloud` GitHub organization. So, I'll wait with the NOTICE.md. Maybe work on CONTRIBUTING.md first, because that is useful help for potential contributors.
